### ISSUE TITLE
Upgrading from 1.61 to 1.6.3 causes error: refuse to serve WS client for illegal ws path: ▒▒I

### ISSUE THREAD

**Issue by nikhilwbhatia** [tsl0922/ttyd#586]
Hi, 

I've been successfully using ttyd version 1.6.1-079aa68 as follows:

```
$ ttyd --port=8004 --base-path=/nbhatia1/project2/ttyd /bin/bash
[2021/01/18 02:13:21:4000] N: ttyd 1.6.1-079aa68 (libwebsockets 4.0.19)
[2021/01/18 02:13:21:4000] N: tty configuration:
[2021/01/18 02:13:21:4000] N:   start command: /bin/bash
[2021/01/18 02:13:21:4000] N:   close signal: SIGHUP (1)
[2021/01/18 02:13:21:4000] N:   terminal type: xterm-256color
[2021/01/18 02:13:21:4000] N: endpoints:
[2021/01/18 02:13:21:4000] N:   base-path: /nbhatia1/project2/ttyd
[2021/01/18 02:13:21:4000] N:   index    : /nbhatia1/project2/ttyd/
[2021/01/18 02:13:21:4000] N:   token    : /nbhatia1/project2/ttyd/token
[2021/01/18 02:13:21:4000] N:   websocket: /nbhatia1/project2/ttyd/ws
[2021/01/18 02:13:21:4044] N:  Using foreign event loop...
[2021/01/18 02:13:21:4044] N:  Listening on port: 8004
[2021/01/18 02:13:23:2686] N: HTTP /nbhatia1/project2/ttyd/token - 127.0.0.1
[2021/01/18 02:13:23:3432] N: WS   /nbhatia1/project2/ttyd/ws - 127.0.0.1, clients: 1
[2021/01/18 02:13:23:3672] N: started process, pid: 156
```

After upgrading to ttyd version 1.6.3-3b174da the same command fails to accept connections:

```
$ ttyd --port=8004 --base-path=/nbhatia1/project2/ttyd /bin/bash
[2021/01/18 02:17:33:1304] N: ttyd 1.6.3-3b174da (libwebsockets 4.1.6-)
[2021/01/18 02:17:33:1305] N: tty configuration:
[2021/01/18 02:17:33:1305] N:   start command: /bin/bash
[2021/01/18 02:17:33:1305] N:   close signal: SIGHUP (1)
[2021/01/18 02:17:33:1305] N:   terminal type: xterm-256color
[2021/01/18 02:17:33:1305] N: endpoints:
[2021/01/18 02:17:33:1305] N:   base-path: /nbhatia1/project2/ttyd
[2021/01/18 02:17:33:1305] N:   index    : /nbhatia1/project2/ttyd/
[2021/01/18 02:17:33:1305] N:   token    : /nbhatia1/project2/ttyd/token
[2021/01/18 02:17:33:1305] N:   websocket: /nbhatia1/project2/ttyd/ws
[2021/01/18 02:17:33:1305] N: LWS: 4.1.6-, loglevel 7
[2021/01/18 02:17:33:1305] N: NET SRV H1 WS IPV6-off
[2021/01/18 02:17:33:1349] N:  Using foreign event loop...
[2021/01/18 02:17:33:1350] N:  Listening on port: 8004
[2021/01/18 02:17:38:0511] N: HTTP /nbhatia1/project2/ttyd/ - 127.0.0.1
[2021/01/18 02:17:38:1701] N: HTTP /nbhatia1/project2/ttyd/token - 127.0.0.1
[2021/01/18 02:17:38:2520] W: refuse to serve WS client for illegal ws path: ▒▒I
[2021/01/18 02:17:38:2520] W: User code denied connection
[2021/01/18 02:17:38:2521] W: refuse to serve WS client for illegal ws path: ▒▒I
[2021/01/18 02:17:38:2521] W: User code denied connection
```

I'm running in a docker container using a CentOS 7 image. Nothing else is changing other than the ttyd version I run (and I can switch between the two versions and replicate the exact same error every time). Could anyone help me debug what's wrong with the websocket client path?

**Comment by tsl0922** [comment]
What if you start ttyd without the `--base-path` option?

**Comment by tsl0922** [comment]
Fixed on master (7bd16386d28cf35a9ead3ca48c31c019ebbdafd5)
